test/Microsoft.Extensions.FileProviders.Composite.Tests/project.json

```diff
@@ -1,9 +1,14 @@
 {
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "compile": "../../shared/**/*.cs"
+  },
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Testing": "1.2.0-*",
     "Microsoft.Extensions.FileProviders.Composite": "1.2.0-*",
-    "Microsoft.Extensions.FileProviders.Sources": "1.2.0-*",
+    "Moq": "4.6.38-alpha",
```

Comment:
4.6.36 is also an alpha. We aren't uniform on this one though. Other repos use the .38 patch.